extensions/github-repository-search/src/index.tsx

```diff
@@ -95,7 +153,7 @@ function Actions(props: { repository: Repository; onVisit: (repository: Reposito
           icon={{ source: "pull-request.png", tintColor: Color.PrimaryText }}
           title="Open Pull Requests"
           url={`${props.repository.url}/pulls`}
-          shortcut={{ modifiers: ["cmd"], key: "p" }}
+          shortcut={{ modifiers: ["cmd", "shift"], key: "p" }}
```

Comment:
cmd+p is the default hotkey to trigger the dropdown so this had to change

Reply:
Makes sense, thanks!